classes/node-red-module.bbclass

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Felix Moessbauer <felix.moessbauer@siemens.com>
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+inherit npm
+inherit node-red-user
```

Comment:
This is the generic class, so 'node-red-user' does not belong here. Instead, add a node-red-module-updatable which build on top of this and makes sure the module gets installed into a writable location.